### DISCUSSION TITLE
general heads up

### DISCUSSION THREAD

**Discussion by christianparpart** [contour-terminal/contour#295] [Development]
Hey @data-man 

A general heads up on what i am currently working/focusing on.

- replace the most important primitive data type uses with strong (boxed) types für better code health and hopefully less bug prone code due to the fact that one can not accidentally swap or misinterpret them. It also helps readability. A draft PR does exist.
- i just today realized i introduced a bug in input mapping so i quickly disabled this feature on master and now i am working on a fix. But to make it good i am refactoring input mapping a bit. That will be merged before the first point.
- whatever is needed to polish the 0.2.0 release. Tbh, i just want to finish it (but I'm a good shape).

**Comment by data-man** [comment]
I 'm working on libunicode+CPM+CI improvements in my private repo.
I don't know how I can update GH workflows in upcoming PR. :(

**Reply by christianparpart** [reply]
I have invited you to the org. That should help maybe?

**Reply by christianparpart** [reply]
You can also self assign tickets on stuff you are working on, if a ticket exists (TBB CI stuff e.g.?)

And yeah, welcome to the team :-)

**Reply by data-man** [reply]
Surprisingly!
Thank you very much!
